jetty-util/src/main/java/org/eclipse/jetty/util/Scanner.java

```diff
@@ -587,8 +619,37 @@ public void removeListener(Listener listener)
     public void doStart() throws Exception
     {
         if (LOG.isDebugEnabled())
-            LOG.debug("Scanner start: rprtExists={}, depth={}, rprtDirs={}, interval={}, filter={}, scannables={}",
-                      _reportExisting, _scanDepth, _reportDirs, _scanInterval, _filter, _scannables);
+            LOG.debug("Scanner start: autoStartScanning={}, reportExists={}, depth={}, rprtDirs={}, interval={}, filter={}, scannables={}",
+                isAutoStartScanning(), _reportExisting, _scanDepth, _reportDirs, _scanInterval, _filter, _scannables);
+
+        // Start the scanner and managed beans (eg: the scheduler)
+        super.doStart();
+
+        if (isAutoStartScanning())
+        {
+            startScanning();
+        }
+    }
+
+    /**
+     * Start the scanning process.
```

Comment:
Don't use the word "process" here.
```suggestion
     * Start scanning.
```